security.md

```diff
@@ -85,6 +86,16 @@ All security advisories made for Foreman are listed below with their correspondi
 
 ### Disclosure details
 
+#### <a id="2021-3584"></a>CVE-2021-3584: Remote code execution through Sendmail configuration
+
+Sendmail location and arguments, available via Administer - Settings, both accept arbitrary strings and pass them into shell. By default, only Foreman super administrators can access settings.
+
+*Mitigation:* Verify the both settings and remove edit_settings permissions from all roles and users until fixed. Alternatively, create settings named `sendmail_location` and `sendmail_arguments` in `settings.yaml` file to override the UI and make the values read-only. Note that the installer will remove these again.
```

Comment:
Just wondering: will users understand that they really should use `:sendmail_location`? Perhaps an example section is good?

Reply:
I still think this is better. Otherwise users may create an entry that ends up being unused. A literal example is even better.

Reply:
Done.